### PR TITLE
Gruvbox Material Theme

### PR DESCRIPTION
**Pull request by KarimLeVallois**

### CODE REVIEW

**Comment by braver** [comment]
We require server version tags, so 1.0.0 instead of 1.0.

**Comment by KarimLeVallois** [comment]
Changed the release to 1.0.0 as requested.

**Comment by braver** [comment]
Technically you require ST > 3092, not simply *. You might want to update your entry to reflect that just to be sure.

**Comment by KarimLeVallois** [comment]
OK, so change the readme to say ST > 3092, rather than just ST 4?

File: repository/g.json [hunk 1]
```diff
@@ -2173,6 +2173,17 @@
 				}
 			]
 		},
+		{
+			"name": "Gruvbox Material Theme",
+			"details": "https://github.com/KarimLeVallois/gruvbox-material-sublime-text",
+			"labels": ["theme", "color scheme"],
+			"releases": [
+				{
+					"sublime_text": "*",
```
Comment (braver) [inline repository/g.json:2182]: \>3092 here instead of *

**Review by braver — CHANGES_REQUESTED** [review]

**Comment by KarimLeVallois** [comment]
Just updated, thanks for your help with this!

**Comment by braver** [comment]
🎉

**Pull request merged**